src/ncp/ncp_base.cpp

```diff
@@ -5167,6 +5169,7 @@ otError NcpBase::SetPropertyHandler_STREAM_NET_INSECURE(uint8_t aHeader, spinel_
 
     (void)aKey;
 
+	OT_UNUSED_VARIABLE(parsedLength);
```

Comment:
Void casting doen'st suppress the warning in IAR when variable is declared and set, but not used. But right, duplication is unnecessary. I will use the macro instead of void casts in these functions.